test/y2partitioner/widgets/format_and_mount_test.rb

```diff
@@ -204,6 +204,50 @@
     include_examples "CWM::ComboBox"
   end
 
+  describe Y2Partitioner::Widgets::EncryptBlkDevice do
+    subject { described_class.new(controller) }
+
+    include_examples "CWM::CheckBox"
+
+    describe "#validate" do
```

Comment:
I find the output a little bit confusing

- encrypt is not selected
  - it returns true
- partition is bigger than 2MiB
  - it returns true
- partition is equal or smaller than 2MiB
  - it returns false
  - it shows error popup

Just imagine we change execution order into:

- partition is equal or smaller than 2MiB
  - it returns false
  - it shows error popup
- encrypt is not selected
  - it returns true
- partition is bigger than 2MiB
  - it returns true

What happens then if the partition is small but encryption is NOT checked? Should I understand it returns false?

My minimum proposed fix:
- encrypt is not selected
  - it returns true
- encrypt is checked and partition is bigger than 2MiB
  - it returns true
- encrypt is checked and partition is equal or smaller than 2MiB
  - it returns false
  - it shows error popup

My preferred fix (I believe that exploring scenarios in width are one of the main advantages of RSpec, as much the produced nesting can be ugly to some people):

- if the partition is bigger than 2MiB
  - and encrypt is not selected
    - it returns true
  - and encrypt is selected
    - it returns true

- if the partition is 2MiB large
  - and encrypt is not selected
    - it returns true
  - and encrypt is selected
    - it returns true

- if the partition is smaller than 2MiB
  - and encrypt is not selected
    - it returns true
  - and encrypt is selected
    - it returns false
    - it shows error popup

Reply:
well, my problem with later one is that so much nesting, but it is so much info that reading it can make you more confuse. Like there is always and encrypt is not selected, can it sometime return false? so minimum proposed fix given better clue and for me is better documentation as it contain less glue and just important info.